src/view/index.js

```diff
@@ -77,4 +85,14 @@ router.route('/forgot-password').post(forgotPassword);
 router.route('/verify-reset-code').post(verifyResetCode);
 router.route('/reset-password').put(resetPassword);
 
+router
+	.route('/courses')
+	.get(adminRoute, handelGetAllCourses)
```

Comment:
courses can be seen/get to all logged in user, so this shouldn't be adminRoute  and it should be secureRoute instead